src/org/jitsi/impl/neomedia/rtp/TransportCCEngine.java

```diff
@@ -145,8 +157,20 @@
     /**
      * Used for estimating the bitrate from RTCP TCC feedback packets
      */
-    private RemoteBitrateEstimatorAbsSendTime bitrateEstimatorAbsSendTime
-        = new RemoteBitrateEstimatorAbsSendTime(this);
+    private final RemoteBitrateEstimatorAbsSendTime bitrateEstimatorAbsSendTime;
+
+    /**
+     * Ctor.
+     *
+     * @param diagnosicContext the {@link DiagnosticContext} of this instance.
+     */
+    public TransportCCEngine(DiagnosticContext diagnosticContext)
+    {
+        this.diagnosticContext
+            = Objects.requireNonNull(diagnosticContext, "diagnosticContext");
```

Comment:
any reason not to use the `@NotNull` annotation?  that's what i thought we settled on (@bgrozev ?)

Reply:
I don't recall any decision regarding `@NotNull`, what's the benefit of using that?